### PR TITLE
Removing FRidh as active maintainer of packages

### PR DESCRIPTION
**Pull request by FRidh**
In the past I was very active with Python packaging. For several years now I was hardly around as maintainer, so it does not make sense I am listed as a maintainer for these makes. Looking back, I should have removed myself as maintainer already much longer ago. Anyway, better late than never.

It's been a fun ride, and  I do intend to occasionally contribute to Nixpkgs, but not in the same way it once was.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/editors/kile/default.nix [hunk 1]
```diff
@@ -42,7 +42,6 @@ stdenv.mkDerivation rec {
   meta = {
     description = "User-friendly TeX/LaTeX authoring tool for the KDE desktop environment";
     homepage = "https://www.kde.org/applications/office/kile/";
-    maintainers = with lib.maintainers; [ fridh ];
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/kile/default.nix:45]: Please keep the maintainers line, sot hat it is obvious that the package has no maintainer.

Comment (FRidh) [reply pkgs/applications/editors/kile/default.nix:45]: The linter does not like that and fails the PR.

Comment (AndersonTorres) [reply pkgs/applications/editors/kile/default.nix:45]: Why?

Comment (mweinelt) [reply pkgs/applications/editors/kile/default.nix:45]: Can we stop with the nitpicking already? It's exhausting.

**Comment by mweinelt** [comment]
Thank you for your contributions and insights, it was a pleasure working with you. Cheers!

File: pkgs/applications/editors/kile/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/applications/editors/kile/default.nix:45]: This was not a blocker from my side, I was legitimately asking why the linter complained about this.

Comment (SuperSandro2000) [reply pkgs/applications/editors/kile/default.nix:45]: @AndersonTorres I am not aware that we are checking this on nixpkgs level, likely some editor linter but I personally think it is fine to ignore that to explicitly document that the package have no maintainer.

**Comment by FRidh** [comment]
> Thank you for your contributions and insights, it was a pleasure working with you. Cheers!

Thanks you too and likewise. It's been a while now, but I enjoyed working together on the Python packages set and staging.

**Pull request merged**